website/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
This is required to update TS to 5.0.

Reply:
I unfortunately had bad experiences with this parameter  in the past, package conflicts without end.

Would it be okay if we stick with ts 4 for now?

Reply:
Done!